### ISSUE TITLE
Cryptic "sort mismatch" error

### ISSUE THREAD

**Issue by Calvin-L** [OpenJML/OpenJML#801]
OpenJMLTest.java:
```java
//@ model import java.math.BigInteger;

public class OpenJMLTest {
    //@ public ghost \seq<BigInteger> elems;
}
```

Running `openjml --esc OpenJMLTest.java` results in

```
error: An error while executing a proof script for <init>: (error "line 475 column 100: invalid function application, sort mismatch on argument at position 2")
  >>>(define-fun BL_102_AfterLabel_15__A1 () Bool (= (select |OpenJMLTest2_elems_161_0___4| |`THIS|) NULL))
1 error
```

I don't know if `OpenJMLTest` should verify or not, but the error certainly doesn't help me figure out what's wrong.

**Comment by davidcok** [comment]
It is an internal bug (the primitive collection types like \seq are still in development).
I reproduced it and made this a test, so it will be addressed "soon".

**Comment by davidcok** [comment]
FIxed in V21-0.3
